src/application-error-boundary/ApplicationErrorBoundary.jsx

```diff
@@ -92,10 +96,12 @@ class ApplicationErrorBoundary extends React.Component {
       const errorDetails = activeError.message.toString();
       const errorText = intl.formatMessage({ id: 'terraApplication.errorBoundary.defaultErrorMessage' }, { errorDetails });
       return (
-        <StatusView
-          variant="error"
-          message={errorText}
-        />
+        <div ref={this.errorBoundaryRef} role="alert" tabIndex="-1">
```

Comment:
Wrapping with div would add a default height. which would avoid error status to be displayed in the centre of the page when page level error boundary is implemented.  we might have to set the height of this div to 100% in that case.

Reply:
Removed the `DIV` and role is now a `customProp`

Changed here 1b2a937569096398257177107586295f33a1a467

Reply:
I am not sure. but i had question like do we need to add this role attribute to status-view itself ? there might be this same accessibility issue in applications using status view. so which they have to fix it by passing the role attribute again similar to this. 

@tbiethman , @ryanthemanuel , @lokesh-0813  your thoughts on this  ?